config/config.go

```diff
@@ -340,7 +336,7 @@ type CertsData []CertData
 func (certs *CertsData) Decode(value string) error {
 	err := json.Unmarshal([]byte(value), certs)
 	if err != nil {
-		log.Error("Error unmarshaling TYK_GW_HTTPSERVEROPTIONS_CERTIFICATES")
+		log.Error("Error unmarshalling TYK_GW_HTTPSERVEROPTIONS_CERTIFICATES")
```

Comment:
@mativm02 @buraksekili I think it would be a good idea to return an error here and print the unmarshal err.

Reply:
Hey @tbuchaillot! Do you mean printing and returning the value of "err"? something like this:
<img width="654" alt="image" src="https://user-images.githubusercontent.com/83959431/185353048-e9a45575-2ee2-4fbd-a2b6-290f5ea81cdd.png">